application/features/VNC.py

```diff
@@ -20,6 +20,7 @@
 
 import os
 import re
+import logging
 import threading
```

Comment:
nit: imports should be sorted
```suggestion
import logging
import os
import re
import threading
```